apis/r/tools/install-tiledb-r.R

```diff
@@ -42,4 +42,13 @@ ctb <- contrib.url(getOption("repos"))
 names(ctb) <- getOption("repos")
 dbr <- db[idx[valid], 'Repository']
 (repos <- names(ctb[ctb %in% dbr]))
+
+# BSPM doesn't respect `repos`
+# Check to see if any repo w/ valid tiledb-r is CRAN
+# If not, turn of BSPM
+cran <- getOption("repos")['CRAN']
+cran[is.na(cran)] <- ""
+if (requireNamespace("bspm", quietly = TRUE) && !any(dbr %in% cran)) {
+  bspm::disable()
```

Comment:
What is 'dbr' here?

Reply:
`dbr` is a vector of repositories that a valid version of tiledb-r is present in. In this case, it's `https://tiledb-inc.r-universe.dev`

See log line at https://github.com/single-cell-data/TileDB-SOMA/actions/runs/9035919602/job/24831666413?pr=2546#step:9:169